### DISCUSSION TITLE
Connection to controller fails after update from 13.10.1 to 13.10.3 or later

### DISCUSSION THREAD

**Discussion by chaosjug** [zwave-js/zwave-js#7504] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
Connection to controller fails after update from 13.10.1 to 13.10.3 or later

What did you expect to happen instead?

update zwave-js

Anything else we should know?
controller is connected via ser2net

### Software versions

Driver (node-zwave-js): 13.10.3 or later

Z-Wave JS UI: 9.26.9 or later

### Z-Wave Controller (Stick/Dongle/...)

Z-Wave.Me 	Z-Wave USB Stick 	UZB
FW: v5.25
SDK: v6.71.1

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

After update: 
[14.3.7.log](https://github.com/user-attachments/files/18203060/14.3.7.log)
Before update
[13.10.1.log](https://github.com/user-attachments/files/18203063/13.10.1.log)

[a comment by an automated account is not shown]

**Comment by chaosjug** [comment]
Stuff about bootloader shows up in the new log:
```
2024-12-17T21:57:19.214Z CNTRLR   [BOOTLOADER] ser2net port tcp,2222 device serialdev, /dev/ttyACM0, 115200n81 [
                                  ] (Debian GNU/Linux)
```
Maybe a regression due to https://github.com/zwave-js/node-zwave-js/pull/7327 ?
This is also the only change mentioned in the release notes for [Release v13.10.2](https://github.com/zwave-js/node-zwave-js/releases/tag/v13.10.2)

**Reply by AlCalzone** [reply, marked as answer]
Yeah this is effectively a regression, but "ignoring" the banner message from ser2net and similar was never supposed to work.

https://github.com/zwave-js/node-zwave-js/issues/7398#issuecomment-2478764253 is the same issue, but for ser2sock.

You'll have to disable the `banner: ...` part in the ser2net config: https://manpages.ubuntu.com/manpages/jammy/man5/ser2net.yaml.5.html#connection%20specification

**Reply by chaosjug** [reply]
That fixed the issue. Thanks!
